components/automate-ui/src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.html

```diff
@@ -69,6 +69,12 @@ <h2 slot="title">Add Chef Organization</h2>
           </chef-error>
         </chef-form-field>
       </div>
+      <div class="input-margin">
+        <app-projects-dropdown
+          (onDropdownClosing)="onProjectDropdownClosing($event)"
+          [projectsUpdated]="projectsUpdatedEvent">
```

Comment:
You will probably need to amend your usage here as in this commit: https://github.com/chef/automate/pull/3698/commits/1c5c200e3fed5d3d8de9e18d37602784c569da4e
We found that if you create a thing in the modal, then re-open the modal, the projects list was not getting reset to "(unassigned)." If you see the same problem (and I think you will) the changes noted in that commit will fix it.

Reply:
@msorens Yes was facing the issue :) incorporated your suggestion, looking good now. :+1: